frontend/convex/README.md

```
@@ -0,0 +1,90 @@
+# Welcome to your Convex functions directory!
+
+Write your Convex functions here.
+See https://docs.convex.dev/functions for more.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)